pages/common/bash.md

```diff
@@ -28,6 +28,10 @@
 
 `bash -s`
 
+- Stop argument processing bash and pass all the following arguments directly to the command:
```

Comment:
Perhaps this will read better?
```suggestion
- Stop bash argument processing and pass all subsequent arguments directly to the script being executed:
```

Reply:
Yes. I would commit your suggested change.